### DISCUSSION TITLE
Markdown (Notes) [Idea from Notion]

### DISCUSSION THREAD

**Discussion by Neuxbane** [nocodb/nocodb#8979] [Features]
Hi, developers... Could you add a simple Markdown like in the Notion application? It would be great if this app has this simple note ❤️. Just make the simple one if the functions in the Notion application are complicated.

Here's the preview from Notion application:
![Screen Shot 2024-07-10 at 17 06 02](https://github.com/nocodb/nocodb/assets/65763323/611b7aaf-c020-4a45-bec6-fdb0cb100aaf)

Best regards, Neuxbane.

**Comment by o1lab** [comment]
Is the idea here to create pages ? 

If not, we do allow to create markdown like text in 'Long Text' field

**Reply by Neuxbane** [reply]
yeah, in Notion it's like creating pages only for Markdown. In the screenshoot there also a link to the database page (GitLab Project List, Gitlab Users List, and Notion PJ Reference)
